tutorials/dapt-curation/code/main.py

```diff
@@ -0,0 +1,238 @@
+# +
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import os
+import shutil
+from typing import Any, Optional, Tuple
+
+import matplotlib.pyplot as plt
+from downloaders import (
+    download_github_sources,
+    download_pdf_sources,
+    download_wikipedia_sources,
+)
+from utils import (
+    clean_and_unify,
+    dedupe,
+    filter_code,
+    filter_code_dataset,
+    filter_code_lines,
+    filter_text,
+    filter_text_lines,
+    redact_code,
+)
+
+import nemo_curator as nc
+from nemo_curator import ScoreFilter, Sequential
+from nemo_curator.datasets import DocumentDataset
+from nemo_curator.filters import RepeatingTopNGramsFilter, WordCountFilter
+from nemo_curator.modifiers.pii_modifier import PiiModifier
+from nemo_curator.modifiers.unicode_reformatter import UnicodeReformatter
+from nemo_curator.modules import ExactDuplicates
+from nemo_curator.modules.modify import Modify
```

Comment:
Nit these should be importable from the top level directly.
```
from nemo_curator import ExactDuplicates, Modify, ScoreFilter, Sequential
```

Reply:
done